bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/config/Device.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.mqtt.homeassistant.internal.config;
+
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+import com.google.gson.annotations.JsonAdapter;
+
+/**
+ * Device configuration
+ *
+ * @author Jochen Klein - Initial contribution
+ */
+@NonNullByDefault
+public class Device {
+    @JsonAdapter(ListOrStringDeserializer.class)
+    protected @Nullable List<String> identifiers;
+    protected @Nullable List<Connection> connections;
+    protected @Nullable String manufacturer;
+    protected @Nullable String model;
+    protected @Nullable String name;
+    protected @Nullable String sw_version;
```

Comment:
This should be camelCase.

Reply:
Will it be OK if I'll rename all these ugly underscore fields at once now?

Reply:
Would be easier for me if you rename only this one now and the others in a separate PR. Otherwise I must review this whole PR again.

Reply:
Field name changed